test/Benchmarks/resnet50_bottleneck.mlir

```diff
@@ -0,0 +1,653 @@
+                %1591 = arith.addf %in, %in_34 : f32
+                linalg.yield %1591 : f32
+    } -> !second_conv1x1_output_tensor_t
+
+    // ReLU
+    %27 = tensor.empty() : !second_conv1x1_output_tensor_t
+    %28 = linalg.generic {
+            indexing_maps = [#map1, #map1, #map1], 
+            iterator_types = ["parallel", "parallel", "parallel", "parallel"]
+        }
+        ins(%26, %cst_9 : !second_conv1x1_output_tensor_t, !second_conv1x1_output_tensor_t)
+        outs(%27 : !second_conv1x1_output_tensor_t) {
+            ^bb0(%in: f32, %in_34: f32, %out: f32):
+                %1591 = arith.maxf %in, %in_34 : f32
+                linalg.yield %1591 : f32
+    } -> !second_conv1x1_output_tensor_t
+    
+    // Tensor reshape for next Conv2D
+    //%1505 = tensor.empty() : second_conv1x1_output_tensor_t
+    //%1506 = linalg.fill ins(%cst_0 : f32) outs(%1505 : second_conv1x1_output_tensor_t) -> second_conv1x1_output_tensor_t
+    //%padded_33 = tensor.pad %28 low[0, 1, 1, 0] high[0, 1, 1, 0] {
+    //^bb0(%arg1: index, %arg2: index, %arg3: index, %arg4: index):
+    //  tensor.yield %cst_0 : f32
+    //} : second_conv1x1_output_tensor_t to tensor<1x9x9x512xf32>
+
+    return %28 : !second_conv1x1_output_tensor_t
+}
+
+func.func @resnet50_bottleneck_block(%input : !first_conv1x1_input_tensor_t) -> !second_conv1x1_output_tensor_t {
+
+    // Call first 1x1 Conv
+    %first_conv1x1_output = call @first_conv2d_1x1_biasadd_bn_relu(%input) : 
+        (!first_conv1x1_input_tensor_t) -> !first_conv1x1_output_tensor_t
+
+    // Pad tensor to feed to Conv 3x3.
+    %cst_0 = arith.constant 0.000000e+00 : f32
+    %padded_first_conv1x1_output = tensor.pad %first_conv1x1_output low[0, 1, 1, 0] high[0, 1, 1, 0] {
+        ^bb0(%arg1: index, %arg2: index, %arg3: index, %arg4: index):
+            tensor.yield %cst_0 : f32
+    } : !first_conv1x1_output_tensor_t to !conv3x3_input_tensor_t
+
+    // Call 3x3 Conv2D
+    %conv3x3_output = call @conv2d_3x3_biasadd_bn_relu(%padded_first_conv1x1_output) : 
+        (!conv3x3_input_tensor_t) -> !conv3x3_output_tensor_t
+
+    // Call 2nd 1x1 Conv2D
+    %second_conv1x1_output = call @second_conv2d_1x1_biasadd_bn_relu(%conv3x3_output) :
+        (!second_conv1x1_input_tensor_t) -> !second_conv1x1_output_tensor_t
+
+    // Skip connection
+    %1 = tensor.empty() : !second_conv1x1_output_tensor_t
+    %2 = linalg.generic {
+            indexing_maps = [#map1, #map1, #map1], 
+            iterator_types = ["parallel", "parallel", "parallel", "parallel"]
+        }
+        ins(%input, %second_conv1x1_output : !first_conv1x1_input_tensor_t, !second_conv1x1_output_tensor_t) 
+        outs(%1 : tensor<1x7x7x2048xf32>) {
+            ^bb0(%in: f32, %in_34: f32, %out: f32):
+                %sum = arith.addf %in, %in_34 : f32
+                linalg.yield %sum : f32
+    } -> !second_conv1x1_output_tensor_t
+
+    return %2 : !second_conv1x1_output_tensor_t
+}
+
+// Output
+// CHECK: ( TBD )
```

Comment:
We have to have checks to be sure the test is doing what it should. Just executing isn't enough to know much.

We need either:
* `FileCheck` looking at the IR output from `tpp-opt`
* `FileCheck` looking at the execution output from `tpp-run`
* Check dialect, comparing the result with a "golden" tensor, and asserting on difference.

Reply:
Using `FileCheck` to check the IR output of `tpp-opt` now.